### ISSUE TITLE
npm install fails when using dprint vscode plugin 

### ISSUE THREAD

**Issue by sheetalkamat** [dprint/dprint#892]
Version: x.x.x - All versions since Typscript code base enabled dprint it has always been the issue, (atleast on windows)

Steps:
Clone https://github.com/Microsoft/TypeScript
Add settings.json in `.vscode` folder by copying template : https://github.com/microsoft/TypeScript/blob/main/.vscode/settings.template.json
open folder in vscode and make sure enable the dprint extension
open any file from `src\compiler`
on command prompt do `npm ci` to see below error.
```
npm error code EPERM
npm error syscall unlink
npm error path c:\TypeScript\node_modules\dprint\dprint.exe
npm error errno -4048
npm error [Error: EPERM: operation not permitted, unlink 'c:\TypeScript\node_modules\dprint\dprint.exe'] {
npm error   errno: -4048,
npm error   code: 'EPERM',
npm error   syscall: 'unlink',
npm error   path: 'c:\\TypeScript\\node_modules\\dprint\\dprint.exe'
npm error }
npm error
npm error The operation was rejected by your operating system.
npm error It's possible that the file was already in use (by a text editor or antivirus),
npm error or that you lack permissions to access it.
npm error
npm error If you believe this might be a permissions issue, please double-check the
npm error permissions of the file and its containing directories, or try running
npm error the command again as root/Administrator.

npm error A complete log of this run can be found in: C:\Users\shkamat\AppData\Local\npm-cache\_logs\2024-07-19T17_38_22_281Z-debug-0.log
```

**Comment by dsherret** [comment]
Sorry about that @sheetalkamat! I noticed that on Windows last week as well.

I think it maybe it should copy the binary from the https://www.npmjs.com/package/@dprint/win32-x64 package to a temporary directory to prevent that issue.

**Comment by dsherret** [comment]
Should be fixed in 0.16.3 (now released)

**Comment by sheetalkamat** [comment]
Thank you so much. its so useful to not have to close vscode or disable the extension after so many months of having to do that
